pkgs/development/web/htmlbeautifier/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, bundlerApp, bundlerUpdateScript, ruby }:
+
+bundlerApp {
+  inherit ruby;
+  pname = "htmlbeautifier";
+  gemfile = ./Gemfile;
+  lockfile = ./Gemfile.lock;
+  gemset = ./gemset.nix;
+  exes = [ "htmlbeautifier" ];
+
+  passthru.updateScript = bundlerUpdateScript "htmlbeautifier";
+
+  meta = with lib; {
+    description = "A normaliser/beautifier for HTML that also understands embedded Ruby, ideal for tidying up Rails templates";
+    homepage = "https://github.com/threedaymonk/htmlbeautifier";
+    license = licenses.mit;
+    platforms = platforms.linux;
```

Comment:
This should probably be `ruby.meta.platforms`, since it's pure Ruby code and should run everywhere Ruby does?